CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.5.0] - 2024-03-21
```

Comment:
Do I have to update this number? Could not find the version txt so used the same number as the previous log

Reply:
resolved